### ISSUE TITLE
Otto fails when attempting to install Terraform

### ISSUE THREAD

**Issue by pvormittag** [hashicorp/otto#365]
Walking through the getting started documentation and during `otto infra`, otto cannot download the needed binaries for terraform. Based on the output, it appears the download URL is broken.  Using Otto 0.1.2 on OSX 10.11.1

```
==> Detecting infrastructure credentials for: otto-getting-started (aws)
    Cached and encrypted infrastructure credentials found.
    Otto will now ask you for the password to decrypt these
    credentials.

Encrypted Credentials Password
  Infrastructure credentials are required for this operation. Otto found
  saved credentials that are password protected. Please enter the password
  to decrypt these credentials. You may also just hit <enter> and leave
  the password blank to force Otto to ask for the credentials again.

  Enter a value:
==> Building main infrastructure...
Would you like Otto to install Terraform?
  Otto requires terraform to be installed, but it couldn't be found on your
  system. Otto can install the latest version of terraform for you. Otto will
  install this into its own private data directory so it doesn't conflict
  with anything else on your system. Would you like Otto to install terraform
  for you? Alternatively, you may install this on your own.

  If you answer yes, Otto will install terraform version 0.6.8.

  Please enter 'yes' to continue. Any other value will exit.

  Enter a value: yes

==> Downloading terraform v0.6.8...
    URL: https://dl.bintray.com/mitchellh/terraform/terraform_0.6.8_darwin_amd64.zip

Error occurred: Error downloading, status code 404
```

**Comment by leowmjw** [comment]
@pvormittag Duplicate of #349; which has a fix in #353.

In the meanwhile until v0.2 is released, you can download the latest binaries and store in the default place Otto looks for (~/.otto.d/binaries).

```
leow$ ls
packer		terraform

leow$ pwd
/Users/leow/.otto.d/binaries
```

**Comment by pvormittag** [comment]
@leowmjw Great, thanks!

**Comment by rmm5t** [comment]
@leowmjw Any idea why putting terraform in `~/.otto.d/binaries` might not work? `otto infra` still prompts me to download terraform (using incorrect download location).

```
$ ls
terraform                         terraform-provider-cloudstack     ...

$ pwd
/Users/rmcgeary/.otto.d/binaries
```

**Comment by leowmjw** [comment]
@rmm5t It looks like you need to put it in a folder named terraform for it to be recognized:

```
host leow$ ls ~/.otto.d/binaries/terraform/
terraform				terraform-provider-digitalocean		terraform-provider-null			terraform-provider-vcd
terraform-provider-atlas		terraform-provider-dme			terraform-provider-openstack		terraform-provider-vsphere
terraform-provider-aws			terraform-provider-dnsimple		terraform-provider-packet		terraform-provisioner-chef
terraform-provider-azure		terraform-provider-docker		terraform-provider-postgresql		terraform-provisioner-file
terraform-provider-azurerm		terraform-provider-dyn			terraform-provider-rundeck		terraform-provisioner-local-exec
terraform-provider-chef			terraform-provider-google		terraform-provider-statuscake		terraform-provisioner-remote-exec
terraform-provider-cloudflare		terraform-provider-heroku		terraform-provider-template
terraform-provider-cloudstack		terraform-provider-mailgun		terraform-provider-terraform
terraform-provider-consul		terraform-provider-mysql		terraform-provider-tls
```

Anyhow, it is now fixed in the stable release v0.2; see below:
```
Ghost leow$ ~/OTTO/bin/otto version
Otto v0.2.0

syamhasan-2:Ghost leow$ ~/OTTO/bin/otto infra
==> Detecting infrastructure credentials for: Ghost (aws)
...
==> Downloading terraform v0.6.9...
    URL: https://releases.hashicorp.com/terraform/0.6.9/terraform_0.6.9_darwin_amd64.zip

```
